modules/tinymce/CHANGELOG.md

```diff
@@ -110,6 +126,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Removed the `editor.settings` property as it's been replaced by the new Options API #TINY-8236
 - Removed the `shortEnded` and `fixed` properties on `tinymce.html.Node` class #TINY-8205
 - Removed the `mceInsertRawHTML` command #TINY-8214
+- Removed the undocumented `editor.editorCommands.hasCustomCommand` API #TINY-7829
+- Removed the undocumented `mceResetDesignMode` command #TINY-7829
+- Removed the undocumented `mceRepaint` command #TINY-7829
+- Removed the undocumented `mceBeginUndoLevel` command #TINY-7829
+- Removed the `execCommand` handler function from the plugin interface #TINY-7829
+- Removed the `execCommand` handler function from the theme interface #TINY-7829
```

Comment:
```suggestion
- Removed the `execCommand` handler function from the plugin and theme interface #TINY-7829
```

Reply:
I like the suggestion but should this be `interfaces`?

Reply:
It should be thanks. I'll edit the suggestion.